packages/boxed-expression-component/src/expressions/ContextExpression/ContextExpression.tsx

```diff
@@ -334,12 +337,20 @@ export function ContextResultInfoCell() {
     return value;
   }, [value]);
 
-  useBeeTableSelectableCellRef(
+  const { isActive } = useBeeTableSelectableCellRef(
     containerCellCoordinates?.rowIndex ?? 0,
     containerCellCoordinates?.columnIndex ?? 0,
     undefined,
     getValue
   );
 
+  const { beeGwtService } = useBoxedExpressionEditor();
+
+  useEffect(() => {
+    if (isActive) {
+      beeGwtService?.selectObject("");
```

Comment:
Essentially, this lines "deselects" what was previously selected, right? Making the properties panel "empty" for when this is selected.

Reply:
Yes.

Reply:
In the Java side, it will try to search for DomainObject with empty UUID which is `NOPDomainObject` (No operation domain object) that is used to empty properties panel.